internal/operators/ocs/validations.go

```diff
@@ -101,11 +75,6 @@ func (o *ocsValidator) ValidateRequirements(cluster *models.Cluster) (api.Valida
 	} else if len(hosts) < o.OCSMasterWorkerHosts { // not supporting OCS installation for 2 Workers and 3 Masters
 		status = "Not supporting OCS Installation for 3 Masters and 2 Workers"
 		o.log.Info("Setting Operator Status", status)
```

Comment:
please change the log to: "Validate Requirements status:"

Reply:
fixed